apps/tehwolfde/src/app/components/app-carousel/app-carousel.component.scss

```
@@ -0,0 +1,68 @@
+  display: flex;
+  gap: 8px;
+}
+
+.carousel-track {
+  display: flex;
+  gap: 24px;
+  margin-top: 16px;
+  overflow-x: auto;
+  scroll-snap-type: x mandatory;
+  scroll-behavior: smooth;
+  // Hide the native scrollbar; the arrow buttons are the affordance.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)